doc/languages-frameworks/python.md

```diff
@@ -804,6 +804,43 @@ If you want to create a Python environment for development, then the recommended
 method is to use `nix-shell`, either with or without the `python.buildEnv`
 function.
 
+### How to consume python modules using pip in a virtualenv like I am used to on other Operating Systems ?
+
+This is an example of a `default.nix` for a `nix-shell`, which allows to consume a `virtualenv` environment,
+and install python modules through `pip` the traditional way.
+
+Create this `default.nix` file, together with a `requirements.txt` and simply execute `nix-shell`.
+
+```
+
+th import <nixpkgs> {};
+with pkgs.python27Packages;
+
+buildPythonPackage { 
+  name = "impurePythonEnv";
+  buildInputs = [
+    taglib
+    openssl
+    git
+    libxml2
+    libxslt
+    libzip
+    python27Full
+    python27Packages.virtualenv
+    python27Packages.pip
+    stdenv
+    zlib ];
```

Comment:
my particular requirements.txt is for a twisted project which requires a few python modules containing C extensions. The packages listed are the dependencies required to build those particular python modules.

I guess I can expand that block stating which are required (like the virtualenv and pip), and which are related to dependencies of the modules listed in the requirements.txt file.